src/svcutilcore/pkg/svcutilcore.pkgproj

```diff
@@ -18,12 +18,18 @@
   <ItemGroup>
     <PackageFile Include=".\files\prefercliruntime"/>
     <PackageFile Include=".\files\dotnet-svcutilcore.runtimeconfig.json">
-      <TargetPath>\lib\netstandard2.0\</TargetPath>
+      <TargetPath>\lib\netcoreapp2.1\</TargetPath>
     </PackageFile>
     <PackageFile Include=".\files\svcutilcore.targets">
       <TargetPath>build\</TargetPath>
     </PackageFile>
   </ItemGroup>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
   <Import Project=".\GenerateNupkgProps.targets" />
+
+    <Target Name="ModifyPackage" AfterTargets="CreatePackage">
+        <Message Text="Call $(WcfRootFolder)UpdateSvcutilCorePackage.ps1 to modify the svcutiolcore.nuspec" Importance="high"/>
+        <Exec Command="powershell set-ExecutionPolicy Unrestricted" />
+        <Exec Command="powershell $(WcfRootFolder)UpdateSvcutilCorePackage.ps1 $(OutputPath)"/>
```

Comment:
I'm pretty certain this is bad as I think you modify the execution policy system wide. Each Exec is a new process launch so this wouldn't work unless this was persisting. Instead modify the second Exec to use the `-ExecutionPolicy UnRestricted` parameter. So it would look something like this:
```xml
<Exec Command="powershell -ExecutionPolicy UnRestricted -File $(WcfRootFolder)UpdateSvcutilCorePackage.ps1 $(OutputPath)"/>
```

Having said that, the UpdateSvcutilCorePackage.ps1 script should be signed and then we don't need to override the ExecutionPolicy.

Reply:
I just realized we don't distribute the script, so ignore the signing idea, but you should override the policy on the command line like my xml snippet shows.

---
In reply to: [188427393](https://github.com/dotnet/wcf/pull/2807#discussion_r188427393) [](ancestors = 188427393)